pkg/trino/client/token.go

```diff
@@ -0,0 +1,24 @@
+package client
+
+import (
+	"time"
+)
+
+const UntilExpirationInSeconds = 60
```

Comment:
This should be higher than the sum of all timeouts of the HTTP request used to fetch the new token. In other words, it should be higher than the maximum time it'll take to fetch a new token. Is that true now? Either way, maybe add a comment explaining how this value was chosen.